### DISCUSSION TITLE
TextDecoderStream not in @types/node

### DISCUSSION THREAD

**Discussion by pauldraper** [DefinitelyTyped/DefinitelyTyped#68623] [Issues with a @types package] [answered]
Added in v16, but not in @types/node.

(And TextEncoderStream, etc)

[a comment by an automated account is not shown]

**Comment by ghost** [comment, marked as answer]
It is present here:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/e8f3d1b317693465ce7d9e49e072dc5da95980a9/types/node/v16/stream/web.d.ts#L353-L388

**Comment by pauldraper** [comment]
Not sure what I saw, thanks.
